### PR TITLE
Low Level Icon link Fix

### LINKED ISSUE #4359: Low level interactions i icon links to old solidity version
the icon here:
image

needs to link to the latest solidity version not to 0.6.2

### PR DESCRIPTION
**Pull request by joeizang**
fixes #4359

### CODE REVIEW

File: libs/remix-ui/run-tab/src/lib/components/universalDappUI.tsx [hunk 1]
```diff
@@ -284,11 +285,12 @@ export function UniversalDappUI(props: UdappProps) {
             <div className="py-2 border-top d-flex justify-content-start flex-grow-1">
               <FormattedMessage id="udapp.lowLevelInteractions" />
             </div>
-            <CustomTooltip placement={'bottom-end'} tooltipClasses="text-wrap" tooltipId="receiveEthDocstoolTip" tooltipText={<FormattedMessage id="udapp.tooltipText8" />}>
+            {/* <CustomTooltip placement={'bottom-end'} tooltipClasses="text-wrap" tooltipId="receiveEthDocstoolTip" tooltipText={<FormattedMessage id="udapp.tooltipText8" />}>
               <a href="https://solidity.readthedocs.io/en/v0.6.2/contracts.html#receive-ether-function" target="_blank" rel="noreferrer">
```
Comment (ryestew) [inline libs/remix-ui/run-tab/src/lib/components/universalDappUI.tsx:289]: should this part be erased?  I think you have it updated in lowLevelInteractionIcon.tsx

**Review by ryestew — COMMENTED** [review]
I'm not sure which is better - specifying the version number or using this url with the "latest" in it:
https://docs.soliditylang.org/en/latest/contracts.html#receive-ether-function

**Comment by joeizang** [comment]
> I'm not sure which is better - specifying the version number or using this url with the "latest" in it: https://docs.soliditylang.org/en/latest/contracts.html#receive-ether-function

It won't matter. This will get the version of the compiler that is loaded.

File: libs/remix-ui/run-tab/src/lib/components/universalDappUI.tsx (hunk 1 above)
Comment (joeizang) [reply libs/remix-ui/run-tab/src/lib/components/universalDappUI.tsx:289]: You are right. This should be nuked.

**Review by ryestew — APPROVED** [review]

**Review by LianaHus — APPROVED** [review]

**Pull request merged**